api/client/diff.go

```diff
@@ -34,7 +34,7 @@ func (cli *DockerCli) CmdDiff(args ...string) error {
 		var kind string
 		switch change.Kind {
 		case archive.ChangeModify:
-			kind = "C"
+			kind = "M"
```

Comment:
Hello, could you tell me whether my PR is in line with the merge standards ? If still have any questions please tell me how to solve it. Since i am a starter, your help or advise is very necessary for me, waiting for your reply, thank you!